stable/database/templates/_helpers.tpl

```diff
@@ -1269,12 +1269,51 @@ Extension point that can be overriden by an embedding chart.
 Database root user
 */}}
 {{- define "database.rootUser" -}}
+{{- if .Values.database.rootUser -}}
 {{ .Values.database.rootUser }}
+{{- else -}}
+{{- $existingValue := ( include "database.findInSecret" (list . ( include "database.secretName" . ) "database-username" )) -}}
+{{- if $existingValue -}}
+{{ $existingValue }}
+dba
```

Comment:
Is `{{- else }}` missing somewhere? It looks like we are printing the current value and also `dba`?